recipe/meta.yaml

```diff
@@ -31,14 +31,13 @@ test:
     requires:
         - pytest
     imports:
-        - pytools
-  commands:
-    - cd $SRC_DIR/test && python -m pytest  # [unix]
-  dev_url: http://github.com/inducer/pytools
+        - {{ name }}
+    commands:
+         - cd $SRC_DIR/test && python -m pytest  # [unix]
 
 
 about:
-    home: http://pypi.python.org/pypi/pytools
+    home: http://pypi.python.org/pypi/{{ name }}
     license: MIT
```

Comment:
Any chance there is a license file in the source that we can include in the package via `license_file`?